keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -163,19 +163,20 @@ _This section must be completed when targeting alpha to a release._
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  No. It's expected to accelerate scheduling a pod upon preemption, and hence it won't impact
+  already running workloads.
 
 * **What specific metrics should inform a rollback?**
+  The existing metric `scheduler_framework_extension_point_duration_seconds{extension_point="Filter"}`,
+  and if the value increase around 5% will inform a rollback.
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
   Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+  The upgrade/rollback and upgrade->downgrade->upgrade paths will be tested manually.
```

Comment:
you need to actually test them before we merge this and if successful we say "successfully tested manually"

Reply:
Done

Reply:
Manually tested this cross different version with feature gate on/off, and passed.